src/extension.ts

```diff
@@ -14,7 +14,7 @@ export async function activate (context: ExtensionContext) {
   Connection.load(context)
 
   client = new Client(context)
-  await client.start()
+  client.start()
```

Comment:
I don't see a change to `client.start` that would make `await`ing it unnecessary. Can you help me understand why this change was made?

Reply:
That was just me appeasing the typescript linter. Message said `'await' has no effect on this type of expression`

Reply:
Ah, so it didn't need to be `await`ed in the first place. Gotcha.